repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/libraries/modscan.py

```diff
@@ -0,0 +1,54 @@
+import os
+import re
+
+from leapp.libraries.stdlib import api
+from leapp.models import RequiredUpgradeInitramPackages, UpgradeDracutModule
+
+_REQUIRED_PACKAGES = [
+    'biosdevname',
+    'binutils',
+    'cifs-utils',
+    'device-mapper-multipath',
+    'dracut',
+    'dracut-config-generic',
+    'dracut-config-rescue',
+    'dracut-network',
+    'dracut-tools',
+    'fcoe-utils',
+    'hostname',
+    'iscsi-initiator-utils',
+    'kbd',
+    'kernel',
+    'kernel-core',
+    'kernel-modules',
+    'keyutils',
+    'lldpad',
+    'lvm2',
+    'mdadm',
+    'nfs-utils',
+    'openssh-clients',
+    'plymouth',
+    'rpcbind',
+    'systemd-container',
+    'tar'
+]
+
+
+def _create_dracut_modules():
+    dracut_base_path = api.get_actor_folder_path('dracut')
```

Comment:
files are only bundled with the actor and not really private - They cannot be discovered by other actors but by specifying the fullpath in the message, their content is accessible. I don't think we need to put them globally since there's no sense for making those files/folders discoverable by all actors.